pkg/cmd/mom/input_resources_command.go

```diff
@@ -0,0 +1,82 @@
+package mom
+
+import (
+	"context"
+
+	"github.com/openshift/multi-operator-manager/pkg/library/libraryinputresources"
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericiooptions"
+)
+
+func NewInputResourcesCommand(streams genericiooptions.IOStreams) *cobra.Command {
+	return libraryinputresources.NewInputResourcesCommand(RunInputResources, streams)
+}
+
+func RunInputResources(ctx context.Context) (*libraryinputresources.InputResources, error) {
+	return &libraryinputresources.InputResources{
```

Comment:
If we get this list wrong then the operator will fail, right ?

Reply:
> If we get this list wrong then the operator will fail, right ?

discussed on the call. to be added.